card/lib/card/env.rb

```diff
@@ -47,7 +47,7 @@ def slot_opts
           opts = params[:slot]&.clone || {}
           opts = opts.to_unsafe_h if opts.is_a? ActionController::Parameters
           opts.merge! shortcut_slot_opts
-          opts.deep_symbolize_keys
+          opts.deep_symbolize_keys.slice(*Card::View::Options.shark_keys)
```

Comment:
This was another emergency patch. It prevents `skip_perms` from being used in params.  It's possible it prevents something we actually want, too.